code/README.md

```diff
@@ -1,13 +1,5 @@
 # Project Labyrinth
 
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
-
-## The problem
-
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
```

Comment:
It seems to be good practice to update the readme. It often gets overlooked but brings a certain flare to the project if it's there.